master/master.cfg

```diff
@@ -107,13 +107,14 @@ c = BuildmasterConfig = {}
 
 c["db_url"] = str(settings.db_url)
 
-# horizons
-c["changeHorizon"] = 300
```

Comment:
What is this setting?

Reply:
As far as I can tell, it hasn't actually done anything since the introduction of the `JanitorConfigurator` below.  I think it was supposed to configure how many source revisions to keep data for.